tools/clang/unittests/HLSL/CompilerTest.cpp

```diff
@@ -1472,6 +1522,7 @@ void CompilerTest::TestPdbUtils(bool bSlim, bool bSourceInDebugModule, bool bStr
       /*HasVersion*/ false,
       /*IsFullPDB*/ true,
       /*hasHasAndPDBName*/true,
+      /*TestReflection*/false,
```

Comment:
And why false here too?  By the way, typo in previous line comment: `hasHasAndPDBName`

Reply:
Fixed.